app/views/homepage/_government_activity.html.erb

```diff
@@ -34,8 +37,15 @@
           text: t("homepage.index.departments_and_organisations"),
         } %>
       </div>
+
+      <%# 
+        The "big number" component links below are hardcoded separately (i.e. not in a loop in this instance) and therefore their 'index' properties are also
+        hardcoded. Additionally, in order to account for these hardcoded components in the 'index_total' property, 2 is added to the ga4_links_counts variable 
```

Comment:
Where is 2 added to the `ga4_links_counts` variable? I'm not sure I understand this comment.

Reply:
It's added in `index.html.erb` - if you'd like to hop on a quick call for me to explain just give me a shout, it's a bit tricky to explain over text